heron/scheduler-core/src/java/com/twitter/heron/scheduler/utils/SchedulerUtils.java

```diff
@@ -127,29 +177,14 @@ private SchedulerUtils() {
    * @param config The static Config
    * @param runtime The runtime Config
    * @param containerIndex the executor/container index
-   * @param freePorts list of free ports
+   * @param ports list of free ports
```

Comment:
The java doc needs to be updated for `ports`

Reply:
will update